src/ResourceManager/DataMigration/Commands.DataMigration/Cmdlets/NewDataMigrationSelectedDB.cs

```diff
@@ -0,0 +1,114 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="NewDataMigrationSelectedDB.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Management.DataMigration.Models;
+
+namespace Microsoft.Azure.Commands.DataMigration.Cmdlets
+{
+    [Cmdlet(VerbsCommon.New, "AzureRmDataMigrationSelectedDB", SupportsShouldProcess = true, DefaultParameterSetName = SqlServerSqlDbParameterSet), OutputType(typeof(MigrateSqlServerSqlDbDatabaseInput))]
```

Comment:
@hitenjava you can remove the `SupportsShouldProcess` property for this cmdlet since you are creating in-memory objects and not making any changes on the server

Reply:
I had it as false earlier, but I got a build failure saying it should be true for 'New' type cmdlets.

Reply:
You can repress that warning, as we have approved this change.